### ISSUE TITLE
Text turns black and is difficult to read.

### ISSUE THREAD

**Issue by KBD20** [07th-mod/matsuribayashi#19]
On the scene where the mountain dogs capture Shion and the screen turns white when Rika decides to climb up, the textbox text goes black and I need to use text history to read.

![Annotation 2020-07-16 165620](https://user-images.githubusercontent.com/46554101/87628550-4d51b680-c785-11ea-8c41-1834b407d8c9.png)

**Comment by ItaloKnox** [comment]
Yeah, we're currently aware of this issue. This particularly affects people
with high opacity like you. A workaround for now is to turn the opacity
down, but I will ask someone to change the colors to something that will
work for any setting.

Em qui, 16 de jul de 2020 01:56, KBD20 <notifications@github.com> escreveu:

**Comment by enumag** [comment]
The best fix in my opinion would be to be able to change the text shadow color as well.
cc @idealpersona @tellowkrinkle

**Comment by tellowkrinkle** [comment]
What commands are causing this?

**Comment by enumag** [comment]
@tellowkrinkle Any `OutputLine` command after [this](https://github.com/07th-mod/matsuribayashi/blob/8d3e3c5376f8f275d3d201239dd8389e11a7a42f/Update/_mats_021.txt#L4848).

**Comment by drojf** [comment]
Not quite as bad as this issue, but there are some readability problems in Japanese (and even in English) when the text is black. See below screenshots:

> this black font in _mats_021.txt doesn't work so well for japanese text, unfortunately

![image](https://user-images.githubusercontent.com/1249449/209737575-5ece363c-0d9c-4fed-828d-ce1120990ea9.png)

![image](https://user-images.githubusercontent.com/1249449/209737578-7c2b50ee-9786-4d42-9378-58643864a977.png)

> a little hard to read even in english too actually

![image](https://user-images.githubusercontent.com/1249449/209737590-a8181791-d621-4795-bcc2-402781d08343.png)


I'm not sure if it is possible, but having a white outline when the text is black would probably resolve these issues.
